### DISCUSSION TITLE
ghostPHEX [Showcase]

### DISCUSSION THREAD

**Discussion by afkvido** [Prodigy-Hacking/ProdigyMathGameHacking#1291] [(D) Developer]
# ghostPHEX

Playing Prodigy at school but your teachers and administrators watch your extensions? And you still want to use PHEX?

This is a perfect solution: the same extension, just with the data modified so it looks like it's Google Docs.

###### This project cooperates with ProdigyMathGameHacking's [LICENSE](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/blob/master/LICENSE). This is a non-monetized, free project.

**Comment by ArcerionDev** [comment]
ok cool but the thing is, you can hide extensions in the extension menu so they don't appear on the search bar

the main issue why people are being found is because of the cheat menu being shown

![image](https://user-images.githubusercontent.com/60945964/150856733-6ae4df8c-5969-480e-8f66-abb666f3f333.png)

**Reply by afkvido** [reply]
the arrow is small tbh, so if its collapsed its fiiiiine

...right?

**Reply by donaldli2020** [reply]
no? it's still pretty suspicious

**Reply by donaldli2020** [reply]
maybe like a keyboard shortcut like alt c (c for cheat) to show the menu

**Reply by afkvido** [reply]
that's a good idea

**Reply by BellaBattilochi** [reply]
Can anyone make me a hacked prodigy account my email is boileghlila@gmail.com
